build.gradle.kts

```diff
@@ -1 +1,16 @@
+plugins {
+    id("io.github.gradle-nexus.publish-plugin") version "1.1.0"
+}
+
 version = "0.12.6"
```

Comment:
Given that we changed the group and artifact IDs I think we can do a major version bump to 1.0.0.

Reply:
done